### PR TITLE
Euen.failing branch

### PR DESCRIPTION
**Pull request by Euen**

### CODE REVIEW

**Comment by gadget-tester** [comment]
**Compiler** failed with exit status: 2

File: src/erlang_repo_sup.erl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-module(erlang-repo_sup).
+-behaviour(supervisor).
+
+-export([start_link/0, init/1]).
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Compiler**:

> function start_link/0 undefined

File: src/erlang_repo_sup.erl [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-module(erlang-repo_sup).
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Compiler**:

> bad module declaration

File: src/erlang_repo_sup.erl [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+-behaviour(supervisor).
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl:2]: According to **Compiler**:

> no module definition

File: src/erlang_repo_sup.erl [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+-export([start_link/0, init/1]).
+
+start_link() ->
+    supervisor:start_link({local, ?MODULE}, ?MODULE, []).
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Compiler**:
> undefined macro 'MODULE'

File: src/erlang_repo_sup.erl [hunk 5]
```diff
@@ -0,0 +1,11 @@
+-module(erlang- repo_sup).
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Compiler**:
> bad module declaration

File: src/erlang_repo_sup.erl [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+-behaviour(supervisor).
+
+-export([start_link/0, init/1]).
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl]: According to **Compiler**:
> function start_link/0 undefined

File: src/erlang_repo_sup.erl [hunk 7]
```diff
@@ -0,0 +1,11 @@
+-module(erlang- repo_sup).
+-behaviour(supervisor).
```
Comment (gadget-tester) [inline src/erlang_repo_sup.erl:2]: According to **Compiler**:

> no module definition

**Comment by gadget-tester** [comment]
**Compiler** failed with exit status: 2Check this for more info.

**Comment by Euen** [comment]
[](url)

File: src/erlang-repo_sup.erl [hunk 8]
```diff
@@ -9,4 +9,4 @@ start_link() ->
 
 init([]) ->
 	Procs = [],
```
Comment (gadget-tester) [inline src/erlang-repo_sup.erl:11]: According to **Elvis**:

> Line 11 has a tab at column 0.

File: src/erlang-repo_sup.erl [hunk 9]
```diff
@@ -9,4 +9,4 @@ start_link() ->
...
-	{ok, {{one_for_one, 1, 5}, Procs}}.
+	{ok, {{one_for_one, 1,5}, Procs}}.
```
Comment (gadget-tester) [inline src/erlang-repo_sup.erl]: According to **Elvis**:

> Missing space after "," on line 12

Comment (gadget-tester) [inline src/erlang-repo_sup.erl]: According to **Elvis**:

> Line 12 has a tab at column 0.

**Comment by Euen** [comment]
f

**Pull request closed without merging**